src/types/accountTypes.ts

```diff
@@ -16,22 +16,20 @@
 
 import type { KeyringPair } from '@polkadot/keyring/types';
 
-import AccountActions from '../actions/accountActions';
+import { AccountActionsTypes } from '../actions/accountActions';
 
-type Account = KeyringPair | null;
+export type Account = KeyringPair | null;
 
 export interface AccountContextType {
   account: Account;
-  receiverAddress?: string;
 }
 
 interface Payload {
-  [propName: string]: any;
+  [propName: string]: Account;
```

Comment:
Now with action creators getting concrete types here is not super important, but it's a nice touch :)